### PR TITLE
Update geosites.txt

### PR DESCRIPTION
**Pull request by frippe12573**
Hi all,

I'm currently working on geosites, and thanks to the many helps I got from the mailing list I propose to modify the README file related to geosites.

### CODE REVIEW

File: docs/tutorials/advanced/geonode_production/geosites.txt [hunk 1]
```diff
@@ -84,6 +83,32 @@ Permissions by Site
...
+    - in /geonode/geonode/contrib/geosites/local_settings.py set the variable SERVE_PATH. it has to point geosites folder.
+    - in the local_setting of the site folder insert the values to the following variables: 
+            * SITE_ID
+            * SITE_NAME
+            * SITE_URL
+6. create static_root directory in your geosites project directory (e.g., /geonode/geonode/contrib/geosites/static_root) and give it grants:
+    sudo mkdir -p ../geonode/geonode/geonode/uploaded/thumbs
+    sudo mkdir -p ../geonode/geonode/geonode/uploaded/layers
+    sudo chown geonode:www-data ../geonode/geonode/contrib/geosites/uploaded/
+    sudo chmod -Rf 777 ../geonode/geonode/contrib/geosites/uploaded/thumbs
```
Comment (simod) [inline docs/tutorials/advanced/geonode_production/geosites.txt]: 777 is not good permission setting, giving ownership to www-data is usually enough. for uploaded/layers the apache configuration should prevent access as well.

File: docs/tutorials/advanced/geonode_production/geosites.txt [hunk 2]
```diff
@@ -84,6 +83,32 @@ Permissions by Site
...
+================
+In order to use geonode in Multi-tenancy mode, follow these steps:
+1. check in settings.py if 'geonode.contrib.geosites' in GEONODE_CONTRIB_APPS is rightly uncommented
+2. add in settings.py 'geonode.contrib.geosites' in INSTALLED_APPS
+3. run python manage.py syncdb
+
 Adding New Sites
 ================
-A management command exists to easily create a new site.  This will create all the needed directories, as well as a site specific settings file.  The command may also create a website configuration file.
+To add a new site follow the following steps:
```
Comment (simod) [inline docs/tutorials/advanced/geonode_production/geosites.txt]: Thanks Eugenio, this PR is good so far. But the following configuration seems to reflect your setup.
The ideal setup is to have geosites outside the geonode code by using a geonode-project for example. It is quite important also to have the statics elsewhere like in /var/www subfolders as apache will serve them

Comment (frippe12573) [inline docs/tutorials/advanced/geonode_production/geosites.txt]: I agree, I modify accordingly

File: docs/tutorials/advanced/geonode_production/geosites.txt (hunk 1 above)
Comment (frippe12573) [inline docs/tutorials/advanced/geonode_production/geosites.txt]: I agree, I modify accordingly

**Comment by frippe12573** [comment]
I followed the comment from Simone. I changed the file accordingly.

File: docs/tutorials/advanced/geonode_production/geosites.txt [hunk 3]
```diff
@@ -84,6 +83,32 @@ Permissions by Site
...
+            * SITE_NAME
+            * SITE_URL
+6. create static_root directory in your geosites project directory (e.g., /geonode/geonode/contrib/geosites/static_root) and give it grants:
+    sudo mkdir -p ../geonode/geonode/geonode/uploaded/thumbs
+    sudo mkdir -p ../geonode/geonode/geonode/uploaded/layers
+    sudo chown geonode:www-data ../geonode/geonode/contrib/geosites/uploaded/
+    sudo chmod -Rf 777 ../geonode/geonode/contrib/geosites/uploaded/thumbs
+    sudo chmod -Rf 777 ../geonode/geonode/contrib/geosites/uploaded/layers
+    sudo chown www-data:www-data ../geonode/geonode/contrib/geosites/uploaded/static_root/
+5. run python manage.py collectstatics - Pay attention on the folder where you are running the command and the folder structure of your geonode/geosites project, in case pass to the path the settings file by using --settings to the python command
```
Comment (gianlucagiacometti) [inline docs/tutorials/advanced/geonode_production/geosites.txt]: collectstatic, not collectstatics

Comment (frippe12573) [inline docs/tutorials/advanced/geonode_production/geosites.txt]: right!

**Pull request merged**